flink/v1.13/flink/src/main/java/org/apache/iceberg/flink/FlinkCatalog.java

```diff
@@ -377,6 +378,23 @@ void createIcebergTable(ObjectPath tablePath, CatalogBaseTable table, boolean ig
     PartitionSpec spec = toPartitionSpec(((CatalogTable) table).getPartitionKeys(), icebergSchema);
 
     ImmutableMap.Builder<String, String> properties = ImmutableMap.builder();
+
+    List<WatermarkSpec> watermarkSpecs = table.getSchema().getWatermarkSpecs();
+    if (watermarkSpecs != null) {
+      for (int i = 0; i < watermarkSpecs.size(); i++) {
+        WatermarkSpec watermarkSpec = watermarkSpecs.get(i);
+        properties.put(
+            DescriptorProperties.WATERMARK + '.' + i + '.' + DescriptorProperties.WATERMARK_ROWTIME,
```

Comment:
since these watermark spec properties are Flink specific, we probably should add a `flink.` prefix.

Reply:
Ok, i will add `flink.` prefix.